pkg/dependencies/version.go

```diff
@@ -0,0 +1,70 @@
+package dependencies
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+
+	"github.com/Masterminds/semver"
+	"github.com/pkg/errors"
+)
+
+// returns the currently installed clang version
+func clangVersion(dep *Dependency) (*semver.Version, error) {
+	path, err := dep.finder.ClangPath()
+	if err != nil {
+		return nil, err
+	}
+
+	re := regexp.MustCompile(`(?m)^clang version (?P<version>\d+.\d+.\d+)$`)
+	version, err := parseCommandVersionWithRegex(path, []string{"--version"}, re, dep.Key)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+	return version, nil
+}
+
+// helper for parsing the --version output for different llvm tools,
+// for example llvm-cov, llvm-symbolizer
+func llvmVersion(path string, dep *Dependency) (*semver.Version, error) {
+	re := regexp.MustCompile(`(?m)^\s*LLVM version (?P<version>\d+.\d+.\d+)$`)
```

Comment:
On my system: `Ubuntu LLVM version 14.0.0`
```suggestion
	re := regexp.MustCompile(`(?m)\s*LLVM version (?P<version>\d+.\d+.\d+)$`)
```